recipes/pytango/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pytango" %}
+{% set version = "9.3.3" %}
+{% set sha256 = "c09fc7a44cac8c2d95d3ad39ba644998e8f468c730dc495443a74de05a07e9f9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cppzmq
+    - cpptango >=9.3,<9.4.0a0
+    - boost
+    - numpy >=1.1
+    - omniorb
+  run:
+    # cpptango set in run_exports
+    - python
+    # boost max_pin set in pin_run_as_build in conda-forge-pinning
+    - boost
+    - {{ pin_compatible('omniorb', min_pin='x.x', max_pin='x.x') }}
+    - {{ pin_compatible('numpy') }}
+    - six >=1.10
+    - enum34  # [py27]
```

Comment:
As Python 2 isn't supported by conda-forge anymore:
```suggestion
```

Reply:
Is it ok to leave it anyway? I'm thinking if someone wants to use this recipe to build it on Python 2 themselves?
Or should we remove it?

Reply:
Sure that's fine 👍